source/_integrations/energenie_power_sockets.markdown

```diff
@@ -0,0 +1,42 @@
+---
+title: Energenie Power-Sockets
+description: Instructions on how to integrate Energenie Power-Strip switches into Home Assistant.
+ha_category:
+  - Switch
+ha_iot_class: Local Polling
+ha_release: '2024.5'
+ha_codeowners:
+  - '@gnumpi'
+ha_domain: energenie_power_sockets
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+The **Energenie Power-Sockets** {% term integration %} allows you to include [Energenie USB Power-Sockets](https://energenie.com/item.aspx?id=7556&lang=de) into your Home Assistant setup.
+
+{% include integrations/config_flow.md %}
+
+## Troubleshooting
+
+If you are running a {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %} installation: Depending on your system configuration, it may be necessary to grant explicit user access rights to the USB device by creating a udev rule.
```

Comment:
```suggestion
If you are running a {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %} installation: Depending on your system configuration, it may be necessary to grant explicit user access rights to the USB device by creating an udev rule.
```